examples/profile/views/pages/profile.ejs

```diff
@@ -39,12 +38,6 @@ const attributes = [
     prop: dateOfBirth,
     icon: "yoti-icon-calendar"
   },
-  {
-    name: "Age verified",
-    preValue: "Age Verification/",
```

Comment:
I think we can get rid of preValue from the example project altogether now, since it was only being used for this.

Would you be able to update the sdk-static-pages repo to remove this, and also to add the AgeVerifications display in? 👍

Reply:
Updated in 22a7aec